models/topics.model.js

```diff
@@ -5,3 +5,38 @@ exports.selectTopics = () => {
         return response.rows;
     });
 };
+
+exports.selectArticles = () => {
+    return db
+        .query(
+            `
+            SELECT
+                articles.author,
+                title,
+                articles.article_id,
+                topic,
+                articles.created_at,
+                articles.votes,
+                CAST (COUNT(comments.article_id) AS INT)
+                    AS comment_count
+            FROM articles
+            LEFT OUTER JOIN comments
+            ON articles.article_id = comments.article_id
+            GROUP BY
+                articles.author,
+                articles.title,
+                articles.created_at,
+                topic,
+                articles.article_id
```

Comment:
You could potentially simplify this a bit, but I wouldn't worry about it for now.